fpsim/locations/README.md

```diff
@@ -7,39 +7,63 @@ This folder stores the location-specific files for FPsim models (both the `<mode
+python fpsim/locations/run_calibrated_location.py nigeria_lagos
+
+# With custom parameters
+python fpsim/locations/run_calibrated_location.py senegal --n-agents 1000 --end-year 2015
+
+# Load existing results instead of running
+python fpsim/locations/run_calibrated_location.py niger --load
+
+# Quick run without saving or generating plots
+python fpsim/locations/run_calibrated_location.py kenya --no-save --no-plots
+```
+
+This script works with any valid FPsim location and provides command-line options for customization.
 
 ## Adding a Location in an (external) Analysis Repo
 
 To add a new location in an analysis repo:
 1. Create a `locations` directory in your analysis repo, and copy into it the following from `fpsim/locations`:
-   2. `locations/template_loc` subdirectory
-   4. `data_utils.py`
-5. In your analysis repo, create a folder `locations/template_loc/data` and add your model data files there (see the 'Model Parameters' section below for the specific files required for an FPsim model to run). Many of the data files have corresponding R processing scripts to format the data, and some are from specific studies and need to be formatted manually. For the proper format of each file, refer to the files in `locations/kenya/data`. **Note that the filenames MUST match the naming conventions defined in the table(s) below.
-6. Change the `template_loc` directory name and `template_loc/template.py` filename to your model location name (must match), and update `template.py` by modifying any lines designated with 'USER-EDITABLE' to your specifications. This file is used to configure a specific FPsim model and generate its location parameter values.
-7. In your analysis repository, you can now use your custom location by adding the following to the top of a script:
-   8. `from locations import <name>`
-   10. `fp.defaults.register_location('<name>', <name>)`
-8. To run fpsim with your location, ensure you also have fpsim installed (either via pip or locally) so that the core model code can be used.
+   - `data_utils.py`
+2. Create a folder `locations/<name>` with a subfolder `locations/<name>/data` and add your model data files there (see the 'Model Parameters' section below for the specific files required). Many of the data files have corresponding R processing scripts to format the data, and some are from specific studies and need to be formatted manually. For the proper format of each file, refer to the files in `locations/kenya/data`. **Note that the filenames MUST match the naming conventions defined in the table(s) below.**
+3. Create `locations/<name>/<name>.py` following the standard pattern with `make_calib_pars()` and `dataloader()` functions (reference existing locations for the exact pattern).
+4. In your analysis repository, you can now use your custom location by adding the following to the top of a script:
+   - `from locations import <name>`
+   - `fp.defaults.register_location('<name>', <name>)`
```

Comment:
This function doesn't exist - users can add a location in an external repo following the example here https://github.com/fpsim/fpsim_senegal